modules/oc-mirror-mirror-to-disk.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-mirroring-disconnected.adoc
+
+[id="oc-mirror-mirror-to-disk_{context}"]
+= Mirroring from a mirror to disk
+
+// TODO: intro
+
+// TODO: usage.md mentioned that this might download hundreds of gigs. And take awhile. Add this warning? And that you can add "log-level=debug" to track progress
+
+.Prerequisites
+
+* You have access to the internet to obtain the necessary container images.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the `oc-mirror` CLI plug-in.
+* You have created the image set configuration file.
+// TODO: Don't need a running cluster, but need some pull secrets. Sync w/ team on this
```

Comment:
@afflom @jpower432 Alex you briefly mentioned that they don't need a running cluster to be able to use oc mirror. But there were some specifics with pull secrets that they did. Can you give me the details on that?

Do they have to do this section (that is in the existing mirroring section), or is it something different: https://docs.openshift.com/container-platform/4.9/installing/installing-mirroring-installation-images.html#installation-adding-registry-pull-secret_installing-mirroring-installation-images

Reply:
Yes, the configuration of the credentials would be the same except for one difference. I believe the currently mirroring process allows users to pass in a custom file with credentials. `oc-mirror` only supports credentials in the default `docker` location. Our goal is to support the default `podman` location as well before tech preview.

Reply:
I will update the procedure to say to use ~/.docker/config.json. Will ping you all for review once I've updated the PR, to see if what I did was correct or not